### PR TITLE
fix: issues dropdown font color

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Change issues dropdown font color.

## Does this PR introduce a user interface change?
- [x] Yes(screenshot is required)
- [ ] No
image
->
image

## Which versions should be patched?
release/1.1

## Which issue(s) this PR fixes:

Fixes # issues dropdown font color.

### CODE REVIEW

File: shell/app/common/components/contractive-filter.scss [hunk 1]
```diff
@@ -102,6 +102,11 @@
     background-color: transparent;
   }
 
+  .ant-dropdown-menu-item,
+  .ant-dropdown-menu-submenu-title {
+    color: $color-dark-65;
+  }
+
```
Comment (McDaddy) [inline shell/app/common/components/contractive-filter.scss:109]: I have one question. Why not set the menu option className?

Comment (hujiahao-hjh) [reply shell/app/common/components/contractive-filter.scss:109]: The previous code has added a ClassName to the Dropdown overlayClassName, so I used that className.

Comment (McDaddy) [reply shell/app/common/components/contractive-filter.scss:109]: I mean why not change className of `Menu.Item`
image

**Pull request merged**